packages/demo/basic/src/graphql/DAOs.ts

```diff
@@ -0,0 +1,17 @@
+import createGovernmentDAO from './government/governmentDAO';
+import createMinistryDAO from './ministry/ministryDAO';
+import createMinisterDAO from './minister/ministerDAO';
+import createVoteDAO from './vote/voteDAO';
+
+function createAllDAOs() {
+  const allDAOs = {
+    governmentDAO: createGovernmentDAO(),
+    ministryDAO: createMinistryDAO(),
+    ministerDAO: createMinisterDAO(),
+    voteDAO: createVoteDAO()
+  };
+
+  return allDAOs;
+}
+
+module.exports = { createAllDAOs };
```

Comment:
any reason why no do just  add export to function createAllDAOs() {

Reply:
Yes, the sinon spy didn't play along ;)